io.catenax.week_based_capacity_group/RELEASE_NOTES.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this model will be documented in this file.
 
+## [3.0.1] - 2024-09-16
```

Comment:
this date should be the date when you got the MS3 approval.

Reply:
changed